addons/google_calendar/models/google_calendar.py

```diff
@@ -350,118 +348,112 @@ def get_one_event_synchro(self, cr, uid, google_id, context=None):
 
         url = "/calendar/v3/calendars/%s/events/%s" % ('primary', google_id)
         try:
-            status, content, ask_time = self.pool['google.service']._do_request(cr, uid, url, params, headers, type='GET', context=context)
+            status, content, ask_time = self.env['google.service']._do_request(url, params, headers, type='GET')
         except:
             _logger.info("Calendar Synchro - In except of get_one_event_synchro")
             pass
 
         return status_response(status) and content or False
 
-    def update_to_google(self, cr, uid, oe_event, google_event, context):
-        calendar_event = self.pool['calendar.event']
+    def update_to_google(self, oe_event, google_event):
+        CalendarEvent = self.env['calendar.event']
 
-        url = "/calendar/v3/calendars/%s/events/%s?fields=%s&access_token=%s" % ('primary', google_event['id'], 'id,updated', self.get_token(cr, uid, context))
+        url = "/calendar/v3/calendars/%s/events/%s?fields=%s&access_token=%s" % ('primary', google_event['id'], 'id,updated', self.get_token())
         headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
-        data = self.generate_data(cr, uid, oe_event, context=context)
+        data = self.generate_data(oe_event)
+        
         data['sequence'] = google_event.get('sequence', 0)
         data_json = simplejson.dumps(data)
 
-        status, content, ask_time = self.pool['google.service']._do_request(cr, uid, url, data_json, headers, type='PATCH', context=context)
+        status, content, ask_time = self.env['google.service']._do_request(url, data_json, headers, type='PATCH')
 
         update_date = datetime.strptime(content['updated'], "%Y-%m-%dT%H:%M:%S.%fz")
-        calendar_event.write(cr, uid, [oe_event.id], {'oe_update_date': update_date})
+        CalendarEvent.browse([oe_event.id]).write({'oe_update_date': update_date})
 
-        if context['curr_attendee']:
-            self.pool['calendar.attendee'].write(cr, uid, [context['curr_attendee']], {'oe_synchro_date': update_date}, context)
+        if self.env.context['curr_attendee']:
```

Comment:
to be on safer side, I would suggest to replace it with self.env.context.get('curr_attendee')